gulpfile.js

```diff
@@ -24,6 +24,11 @@ const rollupIncludePathsPlugin = require('rollup-plugin-includepaths');
 const rollupUglifyPlugin = require('rollup-plugin-uglify');
 const minifier = require('gulp-uglify/minifier');
 const uglifyjs = require('uglify-js');
+const postcss = require('gulp-postcss');
+const postcssImport = require('postcss-import');
+const postcssUrl = require('postcss-url');
+const cssnano = require('cssnano');
```

Comment:
I'm experimenting with another css minifier at the moment, but it seems this one is actively developed, so likely I'm going to recomend cssnano :) At the moment we have relatively tiny css code base so I don't really care.